<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fitness Project</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/images/icon48.png</normaloff>:/new/images/icon48.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCalendarWidget" name="calendarWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="firstDayOfWeek">
           <enum>Qt::Monday</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-65</y>
          <width>760</width>
          <height>476</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Workouts</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="subExer">
                   <property name="text">
                    <string>-</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="addexer">
                   <property name="text">
                    <string>+</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSplitter" name="splitter">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QListWidget" name="exerlist">
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                 <widget class="QFrame" name="frame_51">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="midLineWidth">
                   <number>0</number>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QWidget" name="widget_2" native="true">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_3">
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_11">
                        <item>
                         <widget class="QCheckBox" name="completed">
                          <property name="text">
                           <string>Completed</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <spacer name="horizontalSpacer_9">
                          <property name="orientation">
                           <enum>Qt::Horizontal</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                           <size>
                            <width>40</width>
                            <height>20</height>
                           </size>
                          </property>
                         </spacer>
                        </item>
                        <item>
                         <widget class="QLabel" name="typelabel">
                          <property name="text">
                           <string>type</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLineEdit" name="type"/>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <item>
                         <widget class="QLabel" name="label">
                          <property name="text">
                           <string>distance</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QDoubleSpinBox" name="distance">
                          <property name="maximum">
                           <double>999999.989999999990687</double>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="exerdistu">
                          <property name="text">
                           <string>exerdistu</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <spacer name="horizontalSpacer_2">
                          <property name="orientation">
                           <enum>Qt::Horizontal</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                           <size>
                            <width>40</width>
                            <height>20</height>
                           </size>
                          </property>
                         </spacer>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_3">
                        <item>
                         <widget class="QLabel" name="label_5">
                          <property name="text">
                           <string>energy</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QDoubleSpinBox" name="exerEner">
                          <property name="maximum">
                           <double>999999.989999999990687</double>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="exereneru">
                          <property name="text">
                           <string>exerener</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <spacer name="horizontalSpacer">
                          <property name="orientation">
                           <enum>Qt::Horizontal</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                           <size>
                            <width>40</width>
                            <height>20</height>
                           </size>
                          </property>
                         </spacer>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QTextBrowser" name="description">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="undoRedoEnabled">
                      <bool>true</bool>
                     </property>
                     <property name="readOnly">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_4">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Food</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_7">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="subNom">
                   <property name="text">
                    <string>-</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="addNom">
                   <property name="text">
                    <string>+</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSplitter" name="splitter_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QListWidget" name="nomlist">
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                 <widget class="QFrame" name="frame_3">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Sunken</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <widget class="QWidget" name="widget_3" native="true">
                     <layout class="QVBoxLayout" name="verticalLayout_9">
                      <item>
                       <widget class="QTimeEdit" name="nomTime"/>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_5">
                        <item>
                         <widget class="QLabel" name="label_7">
                          <property name="text">
                           <string>energy</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QDoubleSpinBox" name="nomener">
                          <property name="maximum">
                           <double>999999.989999999990687</double>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="nomeneru">
                          <property name="text">
                           <string>nomennerlab</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QTextBrowser" name="note">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="undoRedoEnabled">
                      <bool>true</bool>
                     </property>
                     <property name="readOnly">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_6">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Weight</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_8">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="subWeight">
                   <property name="text">
                    <string>-</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="addweight">
                   <property name="text">
                    <string>+</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QSplitter" name="splitter_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QListWidget" name="weightlist">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                 <widget class="QFrame" name="frame_5">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Sunken</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QWidget" name="widget_4" native="true">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_10">
                      <item>
                       <widget class="QTimeEdit" name="weighttime"/>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_7">
                        <item>
                         <widget class="QLabel" name="label_8">
                          <property name="text">
                           <string>weight</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QDoubleSpinBox" name="weight">
                          <property name="maximum">
                           <double>9999.989999999999782</double>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="massu">
                          <property name="text">
                           <string>massu</string>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_5">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuDay">
    <property name="title">
     <string>Day</string>
    </property>
    <addaction name="actionView_totals"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDay"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionView_totals">
   <property name="text">
    <string>View totals</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
